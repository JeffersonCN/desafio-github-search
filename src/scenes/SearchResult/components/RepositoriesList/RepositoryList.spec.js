import React from 'react';
import { shallow } from 'enzyme';
import RepositoryList from './';

describe('Testing RepositoryList Component', () => {
    it('shoud render correctly', () => {
        const repos = [{
            name: "Repositório",
            description: "Descrição",
            stargazers_count: 10
        }]

        const wrapper = shallow(
            <RepositoryList repos={repos}/>
        );

        expect(wrapper).toMatchSnapshot();
    });
});