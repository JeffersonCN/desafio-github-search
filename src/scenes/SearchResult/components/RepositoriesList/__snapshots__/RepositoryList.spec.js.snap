// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing RepositoryList Component shoud render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RepositoriesList
    repos={
        Array [
            Object {
              "description": "Descrição",
              "name": "Repositório",
              "stargazers_count": 10,
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <RepositoryInfo
          repository={
                    Object {
                              "description": "Descrição",
                              "name": "Repositório",
                              "stargazers_count": 10,
                            }
          }
/>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "class",
        "props": Object {
          "repository": Object {
            "description": "Descrição",
            "name": "Repositório",
            "stargazers_count": 10,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <RepositoryInfo
            repository={
                        Object {
                                    "description": "Descrição",
                                    "name": "Repositório",
                                    "stargazers_count": 10,
                                  }
            }
/>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "class",
          "props": Object {
            "repository": Object {
              "description": "Descrição",
              "name": "Repositório",
              "stargazers_count": 10,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
